docs/concepts/sidecarSet/README.md

```diff
@@ -98,6 +98,17 @@ status:
   readyPods: 1
   updatedPods: 1
 ```
+
+### Update a SidecarSet
+
+Use ```kubectl edit``` to modify sidecarSet image from "centos:6.7" to "centos:6.8", then you will see matched pods will be updated in place one by one(we only support maxUnavailable=1 for now).
+
+You could use ```kubectl patch sidecarset test-sidecarset --type merge -p '{"spec":{"paused":true}}'``` to pause the update procedure.
+
+If user modify fields other than image in sidecarSet, pod won't get updated until pod is recreated by workload(eg. deploy), which we called "lazy update". 
```

Comment:
modify->modifies 
eg. -> e.g., 
deploy->deployment (or StatefulSet)

Reply:
I will fix it

Reply:
done